protocols/bftcosi/packets.go

```diff
@@ -4,10 +4,24 @@ import (
 	"crypto/sha512"
 	"errors"
 
+	"github.com/dedis/cothority/network"
 	"github.com/dedis/cothority/sda"
 	"github.com/dedis/crypto/abstract"
 )
 
+func init() {
+	for _, i := range []interface{}{
+		BFTSignature{},
+		Announce{},
+		Commitment{},
+		ChallengePrepare{},
+		ChallengeCommit{},
+		Response{},
+	} {
+		network.RegisterPacketType(i)
```

Comment:
Yes I agree it'd useful. But on the other hand we already have many ways to register a packet (with pointers, with uuids, and decode returns pointer or values etc) and I'd prefer to make an issue about `Cleaning network.RegisterXXX methods` and adding this new method inside the issue. 
Is it ok with you ?